python/lsst/ts/watcher/model.py

```diff
@@ -279,6 +307,34 @@ def get_rules(self, name_regex):
             if compiled_re.match(name) is not None
         )
 
+    def make_filtered_topic_wrapper(self, topic, filter_field):
```

Comment:
Can you please add an example how to use make_filtered_topic_wrapper? What I am missing is something like - say you would like to issue an alarm (call a method?) when ESS.sensor[1].value > 50. To do that, you need to.. + an example. Would that be possible?

Reply:
I'll beef up the docs a bit.

However:
* I plan to add an example of an ESS-based rule to the documentation when I add the humidity rule. That's a nice simple rule (much simpler than the dew point depression rule, because it only reads one kind of sensor).
* We don't yet have a use case for a rule directly creating a FilteredTopicWrapper. Rules using ESS data will use filtered _field_ wrappers (FilteredFieldWrapper and IndexedFilteredFieldWrapper). These field wrappers do contain a topic wrapper, but it's an internal detail.